test/testworker.js

```diff
@@ -273,6 +277,7 @@ export default class TestWorker extends EventEmitter {
     var runId = status.runs.pop().runId;
 
     // Return uniform stats on the worker run (fetching common useful things).
-    return await this.fetchTaskStats(taskId, runId);
+    task.taskId = taskId;
```

Comment:
Ok, I found out why the volume cache test was failing after your commits, but passed on master.  This line breaks reusing a task again within the same test.

This "delete task['taskId'];" needs to be added somewhere before posting the second task here:
https://github.com/taskcluster/docker-worker/blob/master/test/integration/container_volume_caching_test.js#L176